### PR TITLE
libgrust: Add changelog, maintainers

### PR DESCRIPTION
**Pull request by CohenArthur**
Asking for a review of the patch itself, but I will be sending it upstream in a later version of [this patchset](https://gcc.gnu.org/pipermail/gcc-rust/2023-September/001222.html)

### CODE REVIEW

**Comment by CohenArthur** [comment]
hmmm, alright. I'll take it out of the commit. P-E-P should (if he wishes!) be maintainer of the entire Rust frontend anyway.

**Review by P-E-P — COMMENTED** [review]
Aren't the changes in https://github.com/Rust-GCC/gccrs/blob/master/contrib/gcc_update#L68 missing?

**Comment by CohenArthur** [comment]
yeah, I didn't do them haha. could you push/rebase them to this branch please @P-E-P ?

File: MAINTAINERS [hunk 1]
```diff
@@ -189,6 +189,7 @@ libquadmath		Tobias Burnus		<tobias@codesourcery.com>
 libquadmath		Jakub Jelinek		<jakub@redhat.com>
 libvtv			Caroline Tice		<cmtice@google.com>
 libphobos		Iain Buclaw		<ibuclaw@gdcproject.org>
+libgrust		All Rust maintainers
```
Comment (tschwinge) [inline MAINTAINERS:192]: Please say "Rust [front end] maintainers", and sort in alphabetically in `lib[...]` (even if not generally done consistently...).

**Pull request merged**